.vale/styles/Meilisearch/Headings.yml

```diff
@@ -51,6 +51,9 @@ exceptions:
   - Bleve & Tantivy
   - Google Cloud Console
   - AMIs
+  - Droplet
+  - Previous
+  - Next
```

Comment:
That's interesting! I'm not surprised that "Next" is throwing a warning, but "Previous" _is_ the first word in the title. Do you think the quotation mark is throwing off the rule?

> "Previous" and "Next" buttons

I wonder if it would be possible to alter the rule so that it ignores capitals immediately following quotation marks. I would prefer that, versus adding these ordinary words to the proper noun list.

Reply:
Good catch! I didn't notice the quotes